### PR TITLE
Add AQI rating template sensor example to the PurpleAir docs

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

Many people ask why the PurpleAir integration doesn't have an AQI sensor (the API doesn't provide that data). I've been sharing an example to create one via a template sensor; this PR includes it in the official docs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bachya** [comment]
Not sure why CI is failing.

**Comment by domoritz** [comment]
This is great. Thanks for adding the example to get the AQI levels. Could you also add an example to calculate the AQI (as a number) from the PM2.5? I think the documentation is at https://www.airnow.gov/sites/default/files/2020-05/aqi-technical-assistance-document-sept2018.pdf and there are some corrections needed. https://gitlab.com/gibwar/home-assistant-purpleair/-/tree/main/ has the calculations built in.

**Comment by bachya** [comment]
> This is great. Thanks for adding the example to get the AQI levels. Could you also add an example to calculate the AQI (as a number) from the PM2.5? I think the documentation is at https://www.airnow.gov/sites/default/files/2020-05/aqi-technical-assistance-document-sept2018.pdf and there are some corrections needed. https://gitlab.com/gibwar/home-assistant-purpleair/-/tree/main/ has the calculations built in.

Those calculations are quite complicated for a template sensor. Doesn't mean it can't be done, but I would recommend that being a separate docs PR.

**Comment by lilyball** [comment]
FWIW as a user I would really love the actual corrected AQI number. The general "good/moderate/bad" rating is not particularly helpful.

**Comment by bachya** [comment]
> FWIW as a user I would really love the actual corrected AQI number. The general "good/moderate/bad" rating is not particularly helpful.

I'm not sure I agree with that sentiment (especially since values like "Good" are already directly correlated with an AQI range). Nevertheless, the example in this PR can be adopted as a framework for producing an AQI value instead (as long as the math is changed).

File: source/_integrations/purpleair.markdown [hunk 1]
```diff
@@ -27,3 +28,116 @@ Add Sensor:
...
+The [`statistics`](/integrations/statistics/) sensors can then be combined into a template
+sensor. Note that this example takes a conservative approach: the "worse" of the two
+values (PM2.5 or PM10.0) is used to determine the overall rating.
+
+<div class='note info'>
+Reminder that the breakpoints used below can be determined from the aforementioned EPA
+guidelines.
+</div>
+
+```yaml
```
Comment (frenck) [inline source/_integrations/purpleair.markdown:96]:
````suggestion
{% raw %}

```yaml
````

File: source/_integrations/purpleair.markdown [hunk 2]
```diff
@@ -27,3 +28,116 @@ Add Sensor:
...
+sensor:
+    - name: Local Outdoor Air Quality
```
Comment (frenck) [inline source/_integrations/purpleair.markdown]:
```suggestion
    - name: "Local Outdoor Air Quality"
```

File: source/_integrations/purpleair.markdown [hunk 3]
```diff
@@ -27,3 +28,116 @@ Add Sensor:
...
+          Unhealthy for sensitive groups
+        {% elif rating == 3 %}
+          Unhealthy
+        {% elif rating == 4 %}
+          Very unhealthy
+        {% else %}
+          Hazardous
+        {% endif %}
+      unique_id: local_outdoor_air_quality
+```
```
Comment (frenck) [inline source/_integrations/purpleair.markdown:145]:
````suggestion
```

{% endraw %}
````

**Review by frenck — APPROVED** [review]
Thanks, @bachya 👍 

../Frenck

**Pull request merged**